### ISSUE TITLE
Tag 1.7.0 can't be installed

### ISSUE THREAD

**Issue by evs-xsarus** [SpotlerSoftware/magento2-mailplus-connector#16]
The version in composer.json is incorrect thus this tag is skipped. Hence, I can't install the module on PHP 7.3.x.

From composer:

```
Reading composer.json of mailplus/mailplus-connector (v1.7.0)
Reading /composer/cache/repo/github.com/SpotlerSoftware/magento2-mailplus-connector/160a7b7b840bc118e9a8f01503ca0e883345ee15 from cache
Skipped tag v1.7.0, tag (1.7.0.0) does not match version (1.6.0.0) in composer.json

```

**Comment by PaulBoss** [comment]
We've just released 1.7.2 with corresponding tag, modules.xml and composer.json.

**Comment by evs-xsarus** [comment]
I can install 1.7.2 properly, tnx.
